tools/tests/apps/meteor-ignore/.meteor/versions

```diff
@@ -0,0 +1,64 @@
+allow-deny@1.1.0-beta.30
```

Comment:
These versions a little weird because I initially created the test app on the `release-1.6` branch.